### DISCUSSION TITLE
dark theme on shares

### DISCUSSION THREAD

**Discussion by pricelessrabbit** [outline/outline#2377] [Self Hosting] [answered]
Hi all,
it is possible to set the dark theme for non authenticated users that accessa document using a shared link?

**Comment by tommoor** [comment, marked as answer]
It is not possible to set this as the person creating the share – the document will be displayed in the viewing users system theme preference, so if they have dark theme turned on it will be used.

**Reply by pricelessrabbit** [reply]
oh thanks really nice :)
